fendermint/vm/interpreter/src/fvm/exec.rs

```diff
@@ -83,6 +83,36 @@ where
             anyhow::bail!("failed to apply block cron message: {}", err);
         }
 
+        // Push the current block hash to the chainmetadata actor
+        //
+        if let Some(block_hash) = state.block_hash() {
+            let params = fvm_ipld_encoding::RawBytes::serialize(
+                fendermint_actor_chainmetadata::PushBlockParams {
+                    epoch: height,
+                    block: block_hash,
+                },
+            )?;
+
+            let msg = FvmMessage {
+                from: system::SYSTEM_ACTOR_ADDR,
+                to: fvm_shared::address::Address::new_id(fendermint_actors::CHAINMETADATA_ACTOR_ID),
+                sequence: height as u64,
```

Comment:
How odd, I have no idea why I passed `height` for a nonce to the `cron` actor :thinking:

Reply:
Should I set this to 0?

Reply:
It's fine, I don't know why I set this, but it should work either way. I don't think it's checked for implicit calls.